.github/workflows/digestabot.yaml

```
@@ -27,3 +27,5 @@ jobs:
     - uses: chainguard-dev/digestabot@4298e530896fb4164f9c89030c561b75b9321149 # v1.0.2
       with:
         token: ${{ steps.octo-sts.outputs.token }}
+        author: chainguardian
+        committer: "101908552+chainguardian@users.noreply.github.com"
```

Hm, given these pull requests are done by octo-sts bot, I would want to see octo-sts bot attribution here.

I'm not sure what is its author and commiter name, but would this work?

```suggestion
        author: "octo-sts[bot]"
        committer: "157150467+octo-sts[bot]@users.noreply.github.com"
```

---

sgtm

---

thanks! PTAL